.github/workflows/nix.yml

```
@@ -360,64 +362,65 @@ jobs:
+              --slave /usr/bin/clang++ clang++ /usr/bin/clang++-${CLANG_VERSION}
             update-alternatives --auto clang
-            pip install --no-cache --break-system-packages "conan<2"
-
+            pip install --no-cache --break-system-packages conan~=${CONAN_VERSION}
         - name: checkout
           uses: actions/checkout@v4
-
-        - name: prepare environment
+        - name: Configure Conan
+          uses: ./.github/actions/conan_configure
+          with:
+            profile: Linux-clang-${{ env.CLANG_VERSION }}-Debug
+        - name: download cache
+          uses: actions/download-artifact@v4
+          with:
+            name: Linux-clang-${{ env.CLANG_VERSION }}-Debug
+        - name: extract cache
           run: |
-            mkdir ${GITHUB_WORKSPACE}/.build
-            echo "SOURCE_DIR=$GITHUB_WORKSPACE" >> $GITHUB_ENV
-            echo "BUILD_DIR=$GITHUB_WORKSPACE/.build" >> $GITHUB_ENV
-            echo "CC=/usr/bin/clang" >> $GITHUB_ENV
-            echo "CXX=/usr/bin/clang++" >> $GITHUB_ENV
-
-        - name: configure Conan
+            conan version >> "$GITHUB_STEP_SUMMARY"
+            tar -xzf conan.tgz -C $(conan config home) .
+        - name: prepare environment
           run: |
-            conan profile new --detect default
-            conan profile update settings.compiler=clang default
-            conan profile update settings.compiler.version=${CLANG_RELEASE} default
-            conan profile update settings.compiler.libcxx=libstdc++11 default
-            conan profile update settings.compiler.cppstd=20 default
-            conan profile update options.rocksdb=False default
-            conan profile update \
-              'conf.tools.build:compiler_executables={"c": "/usr/bin/clang", "cpp": "/usr/bin/clang++"}' default
-            conan profile update 'env.CXXFLAGS="-DBOOST_ASIO_DISABLE_CONCEPTS"' default
-            conan profile update 'conf.tools.build:cxxflags+=["-DBOOST_ASIO_DISABLE_CONCEPTS"]' default
-            conan export external/snappy snappy/1.1.10@
-            conan export external/soci soci/4.0.3@
-
+            build_dir="${GITHUB_WORKSPACE}/.build"
+            source_dir="${GITHUB_WORKSPACE}"
+            mkdir -p "${build_dir}"
+            echo "SOURCE_DIR=${source_dir}" >> $GITHUB_ENV
+            echo "BUILD_DIR=${build_dir}" >> $GITHUB_ENV
+            echo "COMPILER=$COMPILER" >> $GITHUB_ENV
+            echo "CLANG_VERSION=$CLANG_VERSION" >> $GITHUB_ENV
         - name: build dependencies
           run: |
             cd ${BUILD_DIR}
-            conan install ${SOURCE_DIR} \
-              --output-folder ${BUILD_DIR} \
-              --install-folder ${BUILD_DIR} \
-              --build missing \
-              --settings build_type=Debug
-
+            conan install "${SOURCE_DIR}" -of "${BUILD_DIR}" \
+              --profile:a Linux-clang-${{ env.CLANG_VERSION }}-Debug \
+              --options xrpld=True \
```

```suggestion
              --options "&:xrpld=True" \
```

Options should look like this to not produce warnings